### PR TITLE
First Project capstone: Build programming course

### PR DESCRIPTION
**Pull request by mohisa302**
### Conference website
in this capstone:

- Add two pages(about and home)
- Add menu icon and navigation to each page
- Use Dom for the speakers section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,99 @@
...
+### Key Features <a name="key-features"></a>
+
+- **using linters**
+- **both mobile and desktop version**
+- **online avaible**
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[you can see programming course through this link](https://mohisa302.github.io/First-capstone-project/)
```
Comment (leonardodiasb) [inline README.md]: - [x] Currently, the live demo link is not showing your project correctly. Kindly make sure you correctly deployed your page with GitHub Pages.
 - 
image
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+      </section>
+
+      <!-- footer -->
+      <footer class="footer">
+        <img class="logo-img" src="./img/logo.png" alt="logo image" />
+        <p class="contact">
+          2015 Creative Commons Korea.<br />
+          Some Rigths Reserved.
+        </p>
+      </footer>
```
Comment (leonardodiasb) [inline index.html]: - [x] Your footer looks good and similar to the template on the mobile version of the 'About' page, but on the desktop version of the 'Home' page the footer is missing and the desktop of the 'About' page is slightly different with the partner section and white background color when on the template it has a dark background color. Kindly make the changes so the footer looks similar to the template on those pages.
 - Your About desktop:
 - 
image

 - Template About desktop:
 - 
image

 - Your Home desktop:
 - 
image

 - Template Home desktop:
 - 
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+      <section class="speakers">
+        <h2>featured Speakers</h2>
+        <hr class="line1" />
+        <div class="speakersContainer">
+        </div>
+        <button class="btn more">
+          <a href="#">MORE</a>
+          <img src="./img/arrow_down.png" alt="arrow icon" />
+        </button>
+        <a href="#" class="btn less">LESS</a>
```
Comment (leonardodiasb) [inline index.html:127]: - [x] Your featured speakers section is nice and very similar to the template, but on the desktop version all the speakers should appear and the button 'more' and 'less' should only appear on the mobile version. Kindly make some changes to fix that.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+          Full-stack Web Development courses offered through Coursera equip learners with knowledge in building web, hybrid, and mobile solutions; understanding front-end frameworks such as Bootstrap 4 and Angular; implementing NoSQL databases using MongoDB; creating the best user
+           experience regardless of device; the basics of ...
+        </p>
+        <h2 class="contact">
+          Please contact us per Email for any further questions<br />
+          about CC Global Summi 2015!<br />
+          <a href="#">ccsummit@cckorea.org</a>
+        </h2>
+      </section>
+
```
Comment (leonardodiasb) [inline index.html]: - [x] On the mobile version this section is overlapping the 'Global Summit' section. Kindly fix the styling to make a better user experience.
 - [x] It is also missing in the 'About' page the middle section containing the logo of the page on both mobile and desktop versions. Kindly implement this section to meet the template.
 - Your mobile version:
 - 
image

 - Template mobile version:
 - 
image

 - Your desktop version:
 - 
image

 - Template desktop version:
 - 
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+          class="about-img"
+          src="./img/about_2013.png"
+          alt="2013 comferance image"
+        />
+        <img
+          class="about-img"
+          src="./img/about_2011.png"
+          alt="2011 comferance image"
+        />
+      </section>
```
Comment (leonardodiasb) [inline index.html]: - [x] Your desktop version of the 'Global Summit' section is the same as the mobile version. Kindly make sure it follows the template and has the 2 items listed as horizontal row and 2 columns.
 - Your version:
 - 
image

 - Template:
 - 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @mohisa302, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Professional README. ✔️ 

## Aspects to improve ♻️

- [ ] Check the comments under the review.
- [ ] Unfortunately, I could not find any link to a video presenting the features of the project you built on your README or this PR description. Please, record a video with a maximum of 5 minutes describing your project, presenting its features, and highlighting a piece of code you enjoyed and are very proud of creating, and paste the link to the video in this PR description.
- 
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mohisa302) [reply README.md]: done

File: index.html (hunk 3 above)
Comment (mohisa302) [reply index.html:127]: done

File: index.html (hunk 4 above)
Comment (mohisa302) [reply index.html]: done

File: index.html (hunk 5 above)
Comment (mohisa302) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (mohisa302) [reply index.html]: Thanks, for your review!

File: index.html [hunk 6]
```diff
@@ -0,0 +1,309 @@
...
+            <a href="#" class="page1">
+              <img class="logo" src="./img/logo.png" alt="logo image"
+            /></a>
+          </li>
+          <li><a href="#" class="page1">Home</a></li>
+          <li><a href="#" class="about-page">About</a></li>
+          <li><a href="#program" id="home-page">Program</a></li>
+          <li><a href="#partner" id="home-page">Sponsor</a></li>
+          <li class="logo2">CC Compaign</li>
+        </ul>
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [OPTIONAL] Good job so far! It is nice if you add style to your `Navigation menu` elements like [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 👍🏼 

image

**Review by mukhammadrizooff — APPROVED** [review]
Hi @mohisa302, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**